### PR TITLE
Change

### PR DESCRIPTION
**Pull request by Elrolon**
Elrolon

### CODE REVIEW

**Comment by Elrolon** [comment]
🚀

**Comment by Elrolon** [comment]
🤩🚀

**Comment by zimbatm** [comment]
Blocked for spamming. Please respect people's time.

File: docs/hook.md [hunk 1]
```diff
@@ -1,4 +1,6 @@
-# Setup
+# Setup changes 
+Change please
```
Comment (Elrolon) [inline docs/hook.md:2]: Transfer

Comment (Elrolon) [inline docs/hook.md:2]: Transfer

**Review by Elrolon — COMMENTED** [review]
🚀🤩

**Pull request closed without merging**